src/AppBundle/Entity/Report/SubmissionCsvFilter.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace AppBundle\Entity\Report;
+
+use AppBundle\Validator\Constraints as AppAssert;
+
+/**
+ * @AppAssert\StartAndEndDateComparison
+ */
+class SubmissionCsvFilter
```

Comment:
Id put this file within a filters folder somewhere, technically its not an entity

Reply:
Done.